src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -560,7 +560,10 @@ private static bool IsParamsArrayExpanded(SemanticModel semanticModel, SyntaxNod
             }
             else
             {
+#pragma warning disable IDE0007 // Use implicit type - Using 'var' causes "error CS8506: No best type was found for the switch expression"
```

Comment:
@allisonchou I thought you had fixed a similar issue earlier?

Another thing to note - this IDE0007 violation only shows up in the IDE, but does not show up in the build. I am not sure about the reason for it.

Reply:
we def need to file this bug.

Reply:
Filed https://github.com/dotnet/roslyn/issues/44507